src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props

```diff
@@ -11,6 +11,7 @@
     <UseRuntimePackageDisclaimer>true</UseRuntimePackageDisclaimer>
     <!-- This is a native package and doesn't contain any ref/lib assets. -->
     <NoWarn>$(NoWarn);NU5128</NoWarn>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

Comment:
This is similar to https://github.com/dotnet/runtime/blob/e0bed9410065d5e3554cbfc58e1ea1f1115c13e4/src/libraries/Microsoft.Bcl.Cryptography/src/Microsoft.Bcl.Cryptography.csproj#L10

We want to disable API compat check as we do not have anything to compare it it. And the binary runtimes really do not have any API surface.

Reply:
Makes sense, please add this as an xml comment above the line.